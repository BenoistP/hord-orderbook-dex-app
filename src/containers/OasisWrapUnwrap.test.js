/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import Immutable from 'immutable';

import {
  OasisWrapUnwrapWrapper,
  mapStateToProps,
  mapDispatchToProps
} from './OasisWrapUnwrap';
import { shallow } from 'enzyme';

describe('(Container) OasisWrapUnwrap', () => {
  const state = Immutable.fromJS({});
  const initialProps = mapStateToProps(state);
  const initialActions = mapDispatchToProps(x => x);
  const props = {
    ...initialActions,
    ...initialProps
  };

  it('will receive right props', () => {
    expect(initialProps).toMatchSnapshot();
  });


  it('will receive right actions', () => {
    expect(initialActions).toMatchSnapshot();
  });

  it('should render', () => {
    const wrapper = shallow(
      <OasisWrapUnwrapWrapper {...props}/>
    );
    expect(wrapper).toMatchSnapshot();
  });

});
