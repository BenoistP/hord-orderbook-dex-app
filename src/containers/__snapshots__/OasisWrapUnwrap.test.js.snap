// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Container) OasisWrapUnwrap should render 1`] = `
<div>
  WrapUnwrap
</div>
`;

exports[`(Container) OasisWrapUnwrap will receive right actions 1`] = `
Object {
  "actions": Object {},
}
`;

exports[`(Container) OasisWrapUnwrap will receive right props 1`] = `Object {}`;
